docs/deploying.md

```diff
@@ -156,6 +156,34 @@ and cannot be undone. Google Cloud Platform currently allows only one Datastore
 database per project, so you would need to create a new project to change the
 database location.
 
+Next, create the following `index.yaml` file:
+
+```bash
+cat > index.yaml << 'EOF'
+indexes:
```

Comment:
Can we move the content of the file to a separate file, e.g. 
deploy/datastore/index.yaml
and ask users to just use the file?

```
gcloud datastore indexes create deploy/datastore/index.yaml
```

should read from the file.

Reply:
Done.